index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+    <title>Like places</title>
+  </head>
+  <body>
+    <div id = "main">
+      <header>
+        <h1>Мої улюблені місця</h1>
+      </header>
+      <hr>
+      <div id = "lviv">
+        <img src="images/wall2.jpg " width="920px">
+      </div>
+      <div id = "kruivka">
+        <img src="images/Kryivka.jpg" height="230px">
+        <aside>
+          <section>
+            <p>
+              «<b>Криї́вка</b>» — ресторан-кнайпа у Львові. Відкритий у вересні 2007 р. Заклад оздоблений у вигляді польової криївки УПА, знаходиться у підвалі одного з будинків за адресою площа Ринок, 14 (Кам'яниця Венеційська). Кухня — українська, польова. Страви подають у військовому посуді. У ресторані є такі атракції, як тир та вільне користування зброєю. Традиція входу до закладу передбачає знання гасла-пароля Слава Україні - Героям слава, не назвавши який не можна потрапити всередину та перевірку чаркою медовухи для тих, хто старший 21 року.
+              <a href="http://uk.wikipedia.org/wiki/%D0%9A%D1%80%D0%B8%D1%97%D0%B2%D0%BA%D0%B0_%28%D1%80%D0%B5%D1%81%D1%82%D0%BE%D1%80%D0%B0%D0%BD%29">
+            <p>More>></p></a></p>
+          </section>
+        </aside>
+      </div>
+      <div id = "legend-house">
+        <img src="images/legendHouse.jpg" height="250px" width = 450px>
+        <section>
+          <p><b>Дім Легенд</b><br>Кнайпа, лише в твоїй уяві. Будинок сажотруса <br>та його сім'ї,
+            це місце де  живуть львівські легенди.<br><br> <a href = "http://www.fest.lviv.ua/uk/restaurants/dimlegend/">More>></a>
+          </p>
+        </section>
+      </div>
+      <div id = "castle">
+        <img src="images/vkastle.jpg" height="250px" width = 450px>
+          <section>
+              <p>
+                  <b>Висо́кий За́мок</b> (пол. Wysoki Zamek) — замок, збудований під керівництвом Галицько-Волинського князя Лева Даниловича.<br><br>
+              </p>
+             <p><a href = "http://uk.wikipedia.org/wiki/%D0%92%D0%B8%D1%81%D0%BE%D0%BA%D0%B8%D0%B9_%D0%B7%D0%B0%D0%BC%D0%BE%D0%BA">More>></a></p>
+          </section>
+      </div>
+        <div id = "rakovets">
+            <img src="images/rakovet.jpg" height="230px">
+        <aside>
+          <section>
+            <p>
+             <b>Ра́ковець</b> — село в Україні, в Пустомитівському районі Львівської області. Населення становить 207 осіб. Орган місцевого самоврядування — Раковецька сільська рада.
+              <a href="http://vorobus.com/2010/06/dzherelo-u-seli-rakovets.html">
+            <p>More>></p></a></p>
+          </section>
+        </aside>
+      </div>
+        </div>
```

Comment:
@Vovan707 Code related to `<div id = "rakovets">` (as well as nested elements) needs to be reformatted, please take a look.